lib/internal/bootstrap_node.js

```diff
@@ -7,9 +7,13 @@
 
 'use strict';
 
-(function(process) {
+(function(global, process) {
+
+  function startup(global, process) {
+    // Expose the global object as a property on itself
+    // (Allows you to set stuff on `global` from anywhere in JavaScript.)
+    global.global = global;
```

Comment:
For the record: this is just the `global->Set(..., global)` from node.cc moved to JS land.